maps/site53/job/jobs.dm

```diff
@@ -105,8 +108,8 @@
 	supervisors = "the Security Commander"
 	department = "Command"
 	department_flag = COM
-	total_positions = 0
-	spawn_positions = 0
+	total_positions = 1
```

Comment:
Do not enable these yet. I have to add some more things.

Reply:
Task list has been taken care of, re-enabling these.